scripts/bosh-cli.sh

```diff
@@ -17,9 +17,11 @@ trap cleanup EXIT
 echo 'Getting BOSH settings'
 
 BOSH_CA_CERT="$(aws s3 cp "s3://gds-paas-${DEPLOY_ENV}-state/bosh-CA.crt" -)"
-BOSH_IP=$(aws ec2 describe-instances \
-    --filters "Name=tag:deploy_env,Values=${DEPLOY_ENV}" 'Name=tag:instance_group,Values=bosh' \
-    --query 'Reservations[].Instances[].PublicIpAddress' --output text)
+BOSH_IP="$(
+  host -T4 "bosh-external.${SYSTEM_DNS_ZONE_NAME}" \
+    | grep 'has address' \
+    | awk '{print $4}'
+)"
```

Comment:
Can we just do:

```
ssh -lots -of -flags -omg \
  "bosh-external.${SYSTEM_DNS_ZONE_NAME}"
```

?

Reply:
I should read the whole script, force push